graphql/data/schema.js

```diff
@@ -1107,6 +1107,157 @@ const appType = new GraphQLObjectType({
   interfaces: [nodeInterface],
 })
```

Comment:
Add to UserType: `currentMission` (MissionType) and `pastMissions` (array of MissionType).